assets/htmlTemplates/abusers.html

```diff
@@ -23,6 +23,11 @@
                 {{ end }}
             </select>
         </div>
+        <div class="form-group">
+            <label for="selector">Selector</label>
+            <br>
+            <select id="" name="selector"><option value="sender_domain">Sender domain</option><option value="sasl_username">Sasl username</option></select>
```

Comment:
No matter what you write your stuff with, I urge you to manually review your own diffs. Saves time for the both of us.